docs/data/joy/customization/dark-mode/DarkModeByDefault.js

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import { CssVarsProvider } from '@mui/joy/styles';
+import Sheet from '@mui/joy/Sheet';
+import Chip from '@mui/joy/Chip';
+import Typography from '@mui/joy/Typography';
+
+export default function DarkModeByDefault() {
+  return (
+    <CssVarsProvider
+      defaultMode="dark"
+      // these props are specific to this demo, you might not need them in your app.
```

Comment:
A short descriptive of the props could help a lot, so that people don't need to switch to the `CssVarsProvider`'s API to check when they might need them

Reply:
Added!